testdata/golang/light/pascalsTriangle.go

```
@@ -0,0 +1,21 @@
+package light
+
+func pascalsTriangle(n int) (t [][]int) {
+	if n < 1 {
+		return
```

we should return an error here. Java throws an exception.